### ISSUE TITLE
Policies document uses lowercase copyright

### ISSUE THREAD

**Issue by csarven** [w3c/w3process#839]
Minor issue (if you want to call it that):

The old reference to https://www.w3.org/Consortium/Legal/ipr-notice#Copyright now redirects to https://www.w3.org/policies/#Copyright (in the browser). Noting that the redirected document includes `id="copyright"` (lowercase "copyright").

This may well be intended going forward but I just wanted to flag it in case it needs any attention. Perhaps as part of other issues / tasks that take fragment identifiers on redirected into account.

See also:
* https://github.com/search?q=org%3Aw3c https%3A%2F%2Fwww.w3.org%2FConsortium%2FLegal%2Fipr-notice%23Copyright&type=code
* https://github.com/search?q=org%3Aw3c+https%3A%2F%2Fwww.w3.org%2Fpolicies%2F%23Copyright&type=code (shows lowercase use, which is good)

This issue may be better in another w3c repository but I couldn't identify one. Move/close issue as you see fit.

/cc @rigow

**Comment by TallTed** [comment]
Note that the link in your first bullet is broken. It should be https://github.com/search?q=org%3Aw3c+https%3A%2F%2Fwww.w3.org%2FConsortium%2FLegal%2Fipr-notice%23Copyright&type=code (note the `+` which could be a %20 following `q=org%3Aw3c`)

Note that the redirect is from `https://www.w3.org/Consortium/Legal/ipr-notice` to `https://www.w3.org/policies/`; the `#Copyright` is strictly client-side.

If a change is needed, it would probably be to change the link from `https://www.w3.org/Consortium/Legal/ipr-notice#Copyright` to `https://www.w3.org/Consortium/Legal/ipr-notice#copyright` (*or*, probably better, to `https://www.w3.org/policies/#copyright`). It *might* be appropriate to add a second fragment ID in the `https://www.w3.org/policies/` document, such that the client browser lands at the same place for both `#Copyright` and `#copyright`.

All of this would be easier to discuss and address if we had a link to the occurrence of the uppercase `#Copyright` url you encountered in whatever document that was!

**Comment by csarven** [comment]
Yes, I'm aware it is client side which is why I said "in the browser" to avoid that discussion =)

Example references to `https://www.w3.org/Consortium/Legal/ipr-notice#Copyright` published by different software, groups, and time:

* https://www.w3.org/TR/did-core/
* https://www.w3.org/TR/ldn/
* https://www.w3.org/TR/annotation-model/
* https://www.w3.org/TR/odrl-model/
...

Note the diff in https://github.com/w3c/specberus/pull/1783/files where the old is using "Copyright". And yes, the identifier did actually exist, e.g., https://web.archive.org/web/20230501044822/https://www.w3.org/Consortium/Legal/2002/ipr-notice-20021231#Copyright . Interestingly in that archived snapshot, the document includes `id`s for both lowercase and uppercase "c".

Any way.. not sure if there is much to do at this point. The PR aboe seemed to acknowledge that change in a way and perhaps it becomes a non-issue for new documents in a long enough time line. Published documents refererring to "Copyright" will get redirected without the browser scrolling the fragment to view.

Again, just wanted to flag it perhaps this issue belongs somewhere else.
